appcues/src/main/java/com/appcues/debugger/ui/DebuggerExt.kt

```diff
@@ -36,7 +36,8 @@ internal fun EventType?.toResourceId(): Int {
 
 internal fun String.toEventType(): EventType = when (this) {
     AnalyticsEvent.ScreenView.eventName -> SCREEN
-    AnalyticsEvent.SessionStarted.eventName -> SESSION
+    AnalyticsEvent.SessionStarted.eventName,
+    AnalyticsEvent.DeviceUpdated.eventName -> SESSION
```

Comment:
looks like on the iOS side there is a new category `device` for this (and upcoming device_unregistered), would be good to keep consistent https://github.com/appcues/appcues-ios-sdk/pull/502/files#diff-142965116d57ab557fe6f255ae2cd3095c91b606e894a2c27b354d7fd491da9bR136